rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -709,7 +717,7 @@ public boolean removeJob(@HeaderParam("sessionid") String sessionId, @PathParam(
     public String jobServerLog(@HeaderParam("sessionid") String sessionId, @PathParam("jobid") String jobId)
             throws NotConnectedRestException, UnknownJobRestException, PermissionRestException {
         try {
-            Scheduler s = checkAccess(sessionId, "jobs/" + jobId + "/log/server");
+            Scheduler s = checkAccess(sessionId, PATH_JOBS + jobId + "/log/server");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "/log/server" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)